index.bs

```diff
@@ -2290,8 +2396,6 @@ When this method is invoked, the user agent MUST execute the following algorithm
     user without [=user consent|consent=], this step MUST NOT be executed before |lifetimeTimer| has expired. See
     [[#sctn-assertion-privacy]] for details.
 
-During the above process, the user agent SHOULD show some UI to the user to guide them in the process of selecting and
-authorizing an authenticator with which to complete the operation.
```

Comment:
Ah, I see now that it moved to the new paragraph in §5.1.4. Use an Existing Credential to Make an Assertion. I'll leave these comments here in case anyone else wonders the same thing.